lib/cog/command/pipeline_parser.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Cog.Command.PipelineParser do
+  alias Piper.Command.Parser
```

Comment:
Might it be cleaner to add a `return_pipelines` option to `Piper.Command.Parser.parse/2`?